modules/cli/pages/index.adoc

```diff
@@ -47,16 +46,310 @@ $ export PATH=$PATH:$HOME/bin
 $ mv skupper $HOME/bin
 ----
 
+endif::[]
+
+ifdef::product[]
+. Subscribe to the required repositories:
++
+--
+Red Hat Enterprise Linux 7::
+[options="nowrap"]
+----
+$ sudo subscription-manager repos --enable=amq-interconnect-1-for-rhel-7-server-rpms --enable=amq-clients-2-for-rhel-7-server-rpms
+----
+
+Red Hat Enterprise Linux 8::
+[options="nowrap"]
+----
+$ sudo subscription-manager repos --enable=amq-interconnect-1-for-rhel-8-x86_64-rpms --enable=amq-clients-2-for-rhel-8-x86_64-rpms
+----
+--
+
+. Use the `yum` or `dnf` command to install the `qpid-dispatch-router`, `qpid-dispatch-tools`, and `qpid-dispatch-console` packages and their dependencies:
++
+----
+$ sudo yum install skupper
+----
+endif::[]
+
+
 . Verify the installation.
 +
 [subs=attributes+]
 ----
-$ skupper --version
-skupper version {skupper-version}
+$ skupper version
+client version {skupper-version}
```

Comment:
@fgiorgetti  Will there be some difference in output of `skupper version` for downstream?

Reply:
Here is a sample that I got yesterday while testing the CLI:
```
skupper -n skupper-east version
client version                 0.5.2-redhat-interconnect-2.0.0
transport version              registry-proxy.engineering.redhat.com/rh-osbs/interconnect-dev-preview-qdr-rhel8:2.0.0-3 (sha256:81e984fa58e5)
controller version             registry-proxy.engineering.redhat.com/rh-osbs/interconnect-dev-preview-service-controller-rhel8:2.0.0-6 (sha256:5f17e857cbbc)
```